frontend/src/components/note/NoteModal.vue

```diff
@@ -61,6 +95,8 @@ watch(visible, (newValue) => {
   nextTick().then(() => {
     if (newValue && formRef.value) {
       formRef.value.setFieldValue('createdAt', new Date());
```

Comment:
if the 'Date' field is read-only, and we always want it to be the current date/time i question whether it needs to be on the form at all. something i could ask Tyler about i guess